components/stats/CreateChart.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
```

Comment:
`import * as React from 'react';` to stay aligned with everything else in the codebase.

Reply:
Fixed